client/bftclient/include/bftclient/config.h

```diff
@@ -66,6 +66,7 @@ struct ClientConfig {
   uint16_t f_val;
   uint16_t c_val;
   RetryTimeoutConfig retry_timeout_config;
+  bool unified_certs = false;
```

Comment:
Please replace unified_certs by use_unified_certs everywhere below to have a consistent parameter name.

Reply:
made the changes